app/notes/[noteId].tsx

```diff
@@ -0,0 +1,74 @@
+import { useFocusEffect, useLocalSearchParams } from 'expo-router';
+import { useCallback, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { colors } from '../../constants/colors';
+import type { NoteResponseBodyGet } from '../api/[noteId]+api';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: colors.background,
+    padding: 10,
+  },
+  textContainer: {
+    alignItems: 'center',
+    gap: 12,
+  },
+  title: {
+    fontSize: 24,
+    fontFamily: 'Poppins_700Bold',
+    color: colors.text,
+    textAlign: 'center',
+  },
+  textContent: {
+    fontSize: 16,
+    fontFamily: 'Poppins_400Regular',
+    color: colors.textSecondary,
+    textAlign: 'center',
+    marginTop: 8,
+  },
+});
+
+export default function NotePage() {
+  const { noteId } = useLocalSearchParams();
+
+  const [title, setTitle] = useState('');
+  const [textContent, setTextContent] = useState('');
+
+  useFocusEffect(
+    useCallback(() => {
+      async function loadNote() {
+        if (typeof noteId !== 'string') {
+          return;
+        }
+
+        const response = await fetch(`/api/${noteId}`);
```

Comment:
This seems like it's not well structured in a REST style - it should probably be:

```
/app/api/notes/[noteId]+api.ts
/app/api/notes/index+api.ts
```

Same with Guests:

```
/app/api/guests/[guestId]+api.ts
/app/api/guests/index+api.ts
```

Reply:
Fixed [5e63643](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/5e6364303ce95173da41d6c805c49c913293dead)

Reply:
This commit duplicated `guests` so that the `fetch` looks like this now:

```ts
const response = await fetch('/api/guests/guests');
```

If you use `index` as I suggested above, it can stay as `/api/guests`

Reply:
changed filename to `index` 
[c625e1c](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/c625e1c2c21720b15143e755ba652e1a086ad8fe)